visualizer/src/Controls/Toolbar/UndoRedo.js

```diff
@@ -0,0 +1,86 @@
+import MuiButton from '@material-ui/core/Button';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import Tooltip from '@material-ui/core/Tooltip';
+import RedoIcon from '@material-ui/icons/Redo';
+import UndoIcon from '@material-ui/icons/Undo';
+import { useSelector } from '@xstate/react';
+import React from 'react';
+import { useUndo } from '../../ServiceContext';
+
+const useStyles = makeStyles({
+  button: {
+    width: '50%',
+  },
+});
+
+// for adding tooltip to disabled buttons
+// from https://stackoverflow.com/questions/61115913
+const Button = withStyles({
+  root: {
+    padding: 4,
+    '&.Mui-disabled': {
+      pointerEvents: 'auto',
+    },
+  },
```

Comment:
I notice this is copy/pasted a few times. Any way we can import this instead? Or include it in a global/theme or CSS?